cfme/fixtures/v2v_fixtures.py

```diff
@@ -273,20 +277,23 @@ def infra_mapping_default_data(source_provider, provider):
         source_provider: Vmware provider
         provider: Target rhev/OSP provider
     """
+    plan_type = VersionPicker({Version.lowest(): None,
```

Comment:
This logic can be flattened, we've established a 59z breakpoint and no longer need to support it in master.

If this is a change you would like to backport for 59z testing as well, please open a PR to 5.9-breakpoint-patched branch.